mk-files/release.mk

```diff
@@ -179,3 +181,30 @@ update-muckrake:
 	fi
 
 	rm -rf $(DIR)
+
+.PHONY: update-packaging
+update-packaging:
+	$(eval DIR=$(shell mktemp -d))
+	$(eval CLI_RELEASE=$(DIR)/cli-release)
+	$(eval PACKAGING=$(DIR)/packaging)
+
+	git clone git@github.com:confluentinc/cli-release.git $(CLI_RELEASE) && \
+	cd $(CLI_RELEASE) && \
+	version=$$(ls release-notes | $(SED) "s/.json$$//" | sort --version-sort | tail -1) && \
+	git clone git@github.com:confluentinc/packaging.git $(PACKAGING) && \
+	cd $(PACKAGING) && \
+	git fetch --all && \
+	branch="bump-cli" && \
+	base=$$(git branch --remote --format "%(refname:short)" | sed -n "s|^origin/\([1-9][0-9]*\.[0-9][0-9]*\.x\)$$|\1|p" | tail -1) && \
+	git checkout $$base && \
+	git checkout $$branch || git checkout -b $$branch && \
+	$(SED) -i "s|cli_BRANCH=\".*\"|cli_BRANCH=\"$${version}\"|" settings.sh && \
+	$(SED) -i "s|CLI_VERSION=.*|CLI_VERSION=$${version}|" release_testing/bin/smoke_test.sh && \
```

Comment:
It looks like one of these should have a `v` and one should not. This code seems to be setting both to the same value, unless I'm reading it incorrectly.

Reply:
> It looks like one of these should have a v and one should not.

Correct.